kurtosis-devnet/pkg/kurtosis/api/wrappers/wrappers.go

```diff
@@ -26,6 +26,23 @@ type ServiceContextWrapper struct {
 	*services.ServiceContext
 }
 
+// mostly a no-op, to force the values to be typed as interfaces
+func convertPortSpecMap(ports map[string]*services.PortSpec) map[string]interfaces.PortSpec {
+	wrappedPorts := make(map[string]interfaces.PortSpec)
+	for name, port := range ports {
+		wrappedPorts[name] = port
+	}
```

Comment:
Still bugs me how the simplest operations take four lines in go

Reply:
yeah, that one in particular is super annoying, as the type system could really easily make that whole block unnecessary